### PR TITLE
Release 2.9.3-beta1

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming first beta of the v2.9.3 series? Well you've just found it, congratulations!

Other notes:
- This includes the refactor of rebase to use the multi-commit-operation logic.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
---- No new flags.
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
---- No new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.9.3-beta1": [
+      "[Added] Adds syntax highlighting for dart - #12827. Thanks @say25!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Add syntax highlighting for dart - #12827. Thanks @say25!",
```

**Review by billygriffin — CHANGES_REQUESTED** [review]
One super minor thing then I'll approve.

**Review by billygriffin — APPROVED** [review]

**Review by bossman84 — COMMENTED** [review]
Good

**Comment by jungjuheeer** [comment]
app/package.json

**Pull request merged**